### ISSUE TITLE
Audit and remove all `TODO(<User>)` comments in the engine codebase.

### ISSUE THREAD

**Issue by chinmaygarde** [flutter/flutter#130519]

**Comment by matanlurey** [comment]
@chinmaygarde Can you explain this one? I've been adding them too, are we not supposed to?

**Comment by chinmaygarde** [comment]
We usually follow the `// TODO<BugNumber>: Description` format for todos as referencing someones GitHub ID becomes tough if the contributor moves on. And we can then reference the issue as a tech-debt in the issue tracker.

**Comment by matanlurey** [comment]
Hmm, that seems to conflict with our own style guide (and lints in place):

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-checking-in-comments-that-ask-questions

**Comment by chinmaygarde** [comment]
Yeah, the C++ style guide https://google.github.io/styleguide/cppguide.html#TODO_Comments also allows for user names. But with the number of ex-contributors referenced in the code base, we gravitated towards bugs with more context instead.
